.golangci.toml

```diff
@@ -5,8 +5,8 @@ skip-files = ["data/.*\\.gen\\.go", "data/generic_nullable_vector\\.go", "data/g
 min-confidence = 1
 
 [linters-settings.goconst]
-min-len = 2
-min-occurrences = 2
+min-len = 5
+min-occurrences = 5
```

Comment:
Those two are to make those kinds of warnings less sensitive:

```
string X has 2 occurrences, make it a constant (goconst)
```

Those changes bump the min occurrences to 5 and min length to 5.

From the original output, with the previous settings, you can see it's quite sensitive and ends up being too annoying in my opinion.

Note those are the same settings we currently use in core as well:

https://github.com/grafana/grafana/blob/25b30aeb6d4309fcbd3391c104a55346d57023a3/.golangci.toml#L4-L6